ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -84,6 +111,34 @@
     }
   }
 
+  &__header-text {
```

Comment:
I think we need to reduce the spacing around this and at the bottom so it more closely matches the design

Reply:
updated thanks